x-pack/legacy/plugins/lens/public/metric_visualization/metric_visualization.tsx

```diff
@@ -94,7 +97,7 @@ export const metricVisualization: Visualization<State, PersistableState> = {
           groupLabel: i18n.translate('xpack.lens.metric.label', { defaultMessage: 'Metric' }),
           layerId: props.state.layerId,
           accessors: props.state.accessor ? [props.state.accessor] : [],
-          supportsMoreColumns: false,
+          supportsMoreColumns: !props.state.accessor,
```

Comment:
Could you shortly explain how this is related to that change and what it does?

Reply:
This is actually a separate issue. Need to fix separately.